.dockerignore

```diff
@@ -0,0 +1,18 @@
+.env
+.env.template
+.env.test
+
+.github
+.pytest_cache
+.vscode
+.venv
+
+ansible
+
+poetry.toml
+
+README.md
+todoapp.service
+todoapp.socket
+
+Dockerfile
```

Comment:
All the entries here look sensible, but one thing to watch out for is that annoyingly `.dockerignore` uses a slightly different syntax to `.gitignore`; the important take away I focus on is that e.g. for line 6 `.pytest_cache` - in `.gitignore` that would ignore all files/folders with that name, but in `.dockerignore`, it will _only_ ignore those in the root directory; we need to specify `**/.pytest_cache` if we want to capture nested folders of that name

No particular issues here, but worth being aware of!